env0/resource_environment_test.go

```diff
@@ -1899,9 +1899,11 @@ func TestUnitEnvironmentWithoutTemplateResource(t *testing.T) {
 				ErrorRegex: "RetryMeForDestroy.*",
 			},
 		},
-		Type:                 "terraform",
+		Type:                 "terragrunt",
```

Comment:
This impacts an environment with a template.
Wanted to test that use-case as well. In this case the user explicitly mentioned here wants to use "terraform" instead of "opentofu".